deploy/helm/galaxy/templates/deployment.yaml

```diff
@@ -44,10 +44,6 @@ spec:
             - name: http
               containerPort: 8000
               protocol: TCP
-          livenessProbe:
-            httpGet:
-              path: /health
-              port: http
```

Comment:
I do think you should not remove it, my suggestion goes to add it to the api in `galaxy/core/api.py` like this:

```python
    
    @app.get("/live")
    async def live():
        return {"status": "live"}
```

And then change the `path` to `/live`

Keeping a liveness probe k8s can restart the pod automatically if for example a deadlock is found.

Reply:
It's always good practice having the probes, so in my opinion I would put the probe